Doc/library/urllib.parse.rst

```diff
@@ -575,6 +575,16 @@ task isn't already covered by the URL parsing functions above.
    Example: ``unquote_to_bytes('a%26%EF')`` yields ``b'a&\xef'``.
 
 
+.. function:: unquote_to_bytes_plus(string)
```

Comment:
Though other functions don't have `versionadded` directive I think it will be good to add one to this addition. 3.8 beta 1 has been released so I am not sure if this can be merged to 3.8.

Reply:
Thanks, I've updated the docs with the versionadded directive set to 3.9, which seems the most logical.